src/azul/service/manifest_service.py

```diff
@@ -622,8 +639,28 @@ def __init_subclass__(cls) -> None:
             cls.cls_for_format[format] = cls
 
     @classmethod
-    def command_lines(cls, url: str) -> Optional[JSON]:
-        return None
+    def _cmd_exe_quote(cls, s: str) -> str:
+        """
+        Escape a string for insertion into a `cmd.exe` command line
+        """
+        assert '"' not in s, s
+        assert '\\' not in s, s
+        return f'"{s}"'
+
+    @classmethod
+    def command_lines(cls, url: str, file_name: str) -> Optional[JSON]:
+        # A URL longer than 255 chars triggers a bug in curl.exe's
+        # implementation of '-O' on Windows. This is especially surprising
+        # because '-O' doesn't need to parse the URL when '-J' is also passed.
+        # To circumvent the URL parsing bug we provide the file name explicitly
+        # with '-o'.
+        cmd_exe_url = cls._cmd_exe_quote(url)
+        cmd_exe_file_name = cls._cmd_exe_quote(file_name)
+        return {
+            # -L follows redirects; -o writes output to the given file
+            'cmd.exe': f'curl.exe -L -o {cmd_exe_file_name} {cmd_exe_url}',
+            'bash': f'curl -L -o {shlex.quote(file_name)} {shlex.quote(url)}'
```

Comment:
It bugs me that this entry inlines  the call to the quoting function while the bash entry doesn't.

Let's switch to 
```suggestion
            'bash': ' '.join([
                'curl', 
                '-L',  
                …
```

and inline `cmd_exe_url` and `cmd_exe_file_name`

Reply:
I also adopted the long-form of the flags as suggested in [Fix: download of curl manifest fails obscurely from 410 status code (#3127) #3229  PR](https://github.com/DataBiosphere/azul/pull/3229#discussion_r683034885).